### PR TITLE
[rhacs-docs-3.72] Issue in file installing/install-ocp-operator.adoc #52063

### PR DESCRIPTION
**Pull request by YashasviChaurasia**
Version(s):

Issue:

#52063

It says: Alternatively, you can use the Red Hat Advanced Cluster Security for Kubernetes web console to find the link to the RHACS portal by performing the following commands:

The instruction is to find the Red Hat Advanced Cluster Security for Kubernetes web console URL, hence the user can not actually use Red Hat Advanced Cluster Security for Kubernetes web console to find Red Hat Advanced Cluster Security for Kubernetes web console web link. :)

Link to docs preview:

QE review:
No QE is required for this PR

Additional information:

### CODE REVIEW

File: modules/verify-central-install-operator.adoc [hunk 1]
```diff
@@ -26,7 +26,8 @@ $ oc -n stackrox get secret central-htpasswd -o go-template='{{index .data "pass
 ----
 $ oc -n stackrox get route central -o jsonpath="{.status.ingress[0].host}"
 ----
-Alternatively, you can use the {product-title} web console to find the link to the {product-title-short} portal by performing the following commands:
+
+Alternatively, you can use the {ocp} web console to find the link to the RHACS portal by performing the following commands:
```
Comment (kcarmichael08) [inline modules/verify-central-install-operator.adoc:30]: Hi, the second {product-title-short} variable is OK to leave - 

```suggestion
Alternatively, you can use the {ocp} web console to find the link to the {product-title-short} portal by performing the following commands:
```

**Comment by bergerhoffer** [comment]
@YashasviChaurasia After you make the change that Kerry mentioned, you'll have to squash your commits so that there is only 1 commit in the PR before we can merge.

**Comment by bergerhoffer** [comment]
Hi @YashasviChaurasia let us know if you're able to make the adjustments requested above, or if you'd like one of our team members to finish this update. Thanks!

**Comment by gaurav-nelson** [comment]
The changes are already in the main `rhacs-docs-branch`.

**Pull request closed without merging**